### PR TITLE
Simplify venv installation, #closes 676

### PR DESCRIPTION
**Pull request by evgeniiz321**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,28 +1,30 @@
 #!/usr/bin/make -f
 
-.DEFAULT_GOAL := help
+.DEFAULT_GOAL := venv.pytest
```
Comment (roman-khimov) [inline Makefile:3]: You don't need it then.

File: .gitignore [hunk 2]
```diff
@@ -19,7 +19,6 @@ xunit_results.xml
 
 # ignore work directories and setup files
 .setup
-.env
```
Comment (roman-khimov) [inline .gitignore:22]: Don't we use this file for local overrides?

File: Makefile (hunk 1 above)
Comment (evgeniiz321) [reply Makefile]: removed

File: .gitignore (hunk 2 above)
Comment (evgeniiz321) [reply .gitignore:22]: In this repo - no, this file is not used and there is no code that handles it. Probably it was used in the past, but not now.

**Review by roman-khimov — APPROVED** [review]

**Review by cthulhu-rider — COMMENTED** [review]
#closes XXX -> closes #XXX

**Pull request merged**